alvr/client_core/src/c_api.rs

```diff
@@ -865,13 +865,37 @@ pub unsafe extern "C" fn alvr_render_stream_opengl(
                 [
                     StreamViewParams {
                         swapchain_index: left_params.swapchain_index,
-                        reprojection_rotation: from_capi_quat(left_params.reprojection_rotation),
-                        fov: from_capi_fov(left_params.fov),
+                        input_view_params: ViewParams {
+                            pose: Pose {
+                                orientation: Quat::IDENTITY,
+                                position: Vec3::ZERO,
+                            },
+                            fov: from_capi_fov(left_params.fov),
+                        },
+                        output_view_params: ViewParams {
+                            pose: Pose {
+                                orientation: from_capi_quat(left_params.reprojection_rotation),
+                                position: Vec3::ZERO,
+                            },
+                            fov: from_capi_fov(left_params.fov),
+                        },
```

Comment:
Use `Pose::default()` and also reuse the same ViewParams instance (bind to a variable)

Reply:
done, I don't think we can reuse anything though because the left/right view params are different unfortunately (maybe some fancy map stuff could be done but I haven't quite gotten the syntax for those down)